variables.tf

```diff
@@ -42,18 +42,26 @@ variable "load_balancer_ip" {
 }
 
 variable "astronomer_namespace" {
-  default     = "astronomer"
-  type        = string
+  default = "astronomer"
+  type    = string
 }
 
 variable "enable_istio" {
-  default     = "false"
-  type        = string
+  default = "false"
+  type    = string
 }
 
 variable "enable_gvisor" {
-  default     = "false"
-  type        = string
+  default = "false"
+  type    = string
 }
 
+variable "gcp_default_service_account_key" {
+  default = ""
+  type    = string
+}
 
+variable "container_registry_bucket_name" {
```

Comment:
where is this used?

Reply:
This will be added to `locals.tf` when we define our helm values:
```
astronomer:
  registry:
    gcs:
      enabled: true
      bucket: ${var.container_registry_bucket_name}
```
Still need to add that bit to `locals.tf`

Reply:
ok, is this PR WIP?

Reply:
Added now!